src/modules/library.js

```diff
@@ -0,0 +1,62 @@
+import Book from './books.js';
+
+export default class Library {
+  constructor() {
+    this.booksCollection = JSON.parse(localStorage.getItem('booksCollection')) || [];
+    this.booksContainer = document.getElementById('book-list');
+    this.titleInput = document.getElementById('title');
+    this.authorInput = document.getElementById('author');
+    this.addButton = document.getElementById('add-button');
+    this.addButton.addEventListener('click', this.addBookToCollection.bind(this));
+    this.displayBooks();
```

Comment:
- [ ] As pointed out by the previous reviewer, there are some functionality that seems not to work as expected after you factored your code, 
- - Looking at the video below, you can see that the add and remove book functionality works fine, but when your refresh or visit the page for the first time, your list of books are not fetched from the local storage and displayed on the page, You need to correct this issue :+1: :100: 

- Video Reference:

https://github.com/Jessekay/AwesomeBookES6/assets/107632780/4b4aabbc-af9e-4ab1-aa40-dc490d6603aa